src/main/java/com/box/sdk/BoxUserInvite.java

```diff
@@ -0,0 +1,177 @@
+package com.box.sdk;
+
+import java.text.ParseException;
+import java.util.Date;
+
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonValue;
+
+/**
+ * Represents an invite object for {@link BoxUser#inviteUser(String, String)}.
+ */
+public class BoxUserInvite extends BoxJSONObject {
+
+    /**
+     * The API connection to be used by the resource.
+     */
+    private BoxAPIConnection api;
+
+    /**
+     * @see #getStatus()
+     */
+    private String status;
+
+    /**
+     * @see #getID()
+     */
+    private String id;
+
+    /**
+     * @see #getInvitedTo()
+     */
+    private BoxEnterprise invitedTo;
+
+    /**
+     * @see #getActionableBy()
+     */
+    private BoxUser.Info actionableBy;
+
+    /**
+     * @see #getInvitedBy()
+     */
+    private BoxUser.Info invitedBy;
+
+    /**
+     * @see #getCreatedAt()
+     */
+    private Date createdAt;
+
+    /**
+     * @see #getModifiedAt()
+     */
+    private Date modifiedAt;
+
+    /**
+     * Constructs an empty Invite.
+     *
+     * @param api the API connection to be used by the resource.
+     */
+    public BoxUserInvite(BoxAPIConnection api) {
+        this.api = api;
+    }
+
+    /**
+     * Constructs an Invite from a JSON string.
+     *
+     * @param api  the API connection to be used by the resource.
+     * @param json the json encoded user invite.
+     */
+    public BoxUserInvite(BoxAPIConnection api, String json) {
+        super(json);
+        this.api = api;
+    }
+

```

Comment:
Add Java Docs
    /**
     * Constructs an Invite from a JSON string.
     *
     * @param api  the API connection to be used by the resource.
     * @param json the JOSNString encoded user invite.
     */